### PR TITLE
Mouse move restrict

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: client/X11/xf_graphics.c [hunk 1]
```diff
@@ -575,6 +575,11 @@ static BOOL xf_Pointer_SetPosition(rdpContext* context, UINT32 x, UINT32 y)
 	WLog_DBG(TAG, "%" PRIu32 "x%" PRIu32, x, y);
 	if (xfc->remote_app && !xfc->focused)
 		return TRUE;
+	else if (!xfc->remote_app)
+	{
+		if (!xfc->focused)
```
Comment (pnowack) [inline client/X11/xf_graphics.c:580]: If we take the already existing `if ()` part here, couldn't we simplify the condition to `if (!xfc->focused && (xfc->remote_app ||  !xfc->remote_app))`, and that to `if (!xfc->focused && (TRUE))`, and that to `if (!xfc->focused)`?

Comment (akallabeth) [reply client/X11/xf_graphics.c:580]: good point

File: client/common/cmdline.c [hunk 2]
```diff
@@ -4001,6 +4001,42 @@ static int freerdp_client_settings_parse_command_line_arguments_int(rdpSettings*
...
+					const PARSE_ON_OFF_RESULT bval = parse_on_off_option(cur);
+					if (bval == PARSE_FAIL)
+						rc = COMMAND_LINE_ERROR_UNEXPECTED_VALUE;
+					else
+					{
+						const BOOL val = bval != PARSE_OFF;
+						size_t key = 0;
+						if (option_starts_with("relative", cur))
+							key = FreeRDP_MouseUseRelativeMove;
+						else if (option_starts_with("grab", cur))
```
Comment (pnowack) [inline client/common/cmdline.c:4024]: Looks like the `local-move` setting is missing

**Review by pnowack — APPROVED** [review]
Just one thing, otherwise lgtm

File: client/common/cmdline.c (hunk 2 above)
Comment (akallabeth) [reply client/common/cmdline.c:4024]: oops, should not have been in header, fixed.

**Comment by akallabeth** [comment]
@freerdp-bot test properly please

**Pull request merged**